eng/Version.Details.xml

```diff
@@ -3,17 +3,17 @@
   <ProductDependencies>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.RemoteExecutor" Version="7.0.0-beta.23478.1">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.23531.4">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>6d7ba5913e94d6f3988f5c66fc57eb00e28b8561</Sha>
+      <Sha>16a344fc2ff2eecac5b03b88c96e99d7796e5125</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.XUnitExtensions" Version="7.0.0-beta.23478.1">
+    <Dependency Name="Microsoft.DotNet.RemoteExecutor" Version="9.0.0-beta.23531.4">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>6d7ba5913e94d6f3988f5c66fc57eb00e28b8561</Sha>
+      <Sha>16a344fc2ff2eecac5b03b88c96e99d7796e5125</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.23478.1">
+    <Dependency Name="Microsoft.DotNet.XUnitExtensions" Version="9.0.0-beta.23531.4">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>6d7ba5913e94d6f3988f5c66fc57eb00e28b8561</Sha>
+      <Sha>16a344fc2ff2eecac5b03b88c96e99d7796e5125</Sha>
```

Comment:
Why did this commit have additional updates to this file? I ask because the previous commit was the darc update, so I would've expected it to take care of these changes already.

Reply:
Not the previous commit, the commit after this one is the Arcade dependency update.

Reply:
This change here just re-orders the entries and brings them manually to the same version.